docs/messaging/messaging-spans.md

```diff
@@ -206,6 +206,7 @@ The following operations related to messages are defined for these semantic conv
 | `create`       | A message is created. "Create" spans always refer to a single message and are used to provide a unique creation context for messages in batch publishing scenarios. |
 | `receive`      | One or more messages are requested by a consumer. This operation refers to pull-based scenarios, where consumers explicitly call methods of messaging SDKs to receive messages. |
 | `deliver`      | One or more messages are passed to a consumer. This operation refers to push-based scenarios, where consumer register callbacks which get called by messaging SDKs. |
+| `settle`       | One or more messages are settled. |
```

Comment:
I think we also need a special-case for settlement span name.

Currently it SHOULD follow `<destination name> <operation name>` pattern, but 

`settle my_queue` is not quite descriptive if more than one settlement kind is supported.
I'd prefer to relax SHOULD and allow systems define how their settlement operations are named.

E.g. RabbitMQ should be able to call them `<destination> ack`  and `<destination> nack` and kafka can use `<destination> commit`. 

I don't see a good reason not to extend it to other spans as well and suggest changing span name part to something like:

```diff
- The values allowed for <operation name> are defined in the section [Operation names](https://github.com/open-telemetry/semantic-conventions/blob/68a87a51e5f4cb3fac9ff029227b73789b462cd7/docs/messaging/messaging-spans.md#operation-names) below. If the format above is used, the operation name MUST match the messaging.operation attribute defined for message consumer spans below.
+ Suggested <operation name> values are defined in the section [Operation names](https://github.com/open-telemetry/semantic-conventions/blob/68a87a51e5f4cb3fac9ff029227b73789b462cd7/docs/messaging/messaging-spans.md#operation-names) below. Messaging systems MAY use their own terminology in span names. For example, RabbitMQ deliver span should be called `<destination> consume` and settle span should be `<destination> ack` or `<destination> nack`
```

We probably need to define span names in extensions for each system.
But this way
- we still have unified `messaging.operation` 
- user-visible name that matches messaging system terminology and API they familiar with 
- we can express settlement type without defining the attribute for it (yet).

Reply:
Discussed in the messaging workgroup: instrumentation SHOULD use `<destination name> <operation name>`. If a messaging system wants to use a different name, they MUST document it in messaging specific conventions.

Reply:
I removed this sentence:
> If the format above is used, the operation name MUST match the `messaging.operation` attribute defined for message consumer spans below.

And I added this one:
> Messaging system specific adaptions to span naming MUST be documented in [semantic conventions for specific messaging technologies](#semantic-conventions-for-specific-messaging-technologies).

Please have another look @open-telemetry/semconv-messaging-approvers